.devcontainer/Dockerfile

```
@@ -14,8 +14,9 @@ RUN if [ "${NODE_VERSION}" != "none" ]; then su vscode -c "umask 0002 && . /usr/
 #    && rm -rf /tmp/pip-tmp
 
 # [Optional] Uncomment this section to install additional OS packages.
-# RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
-#     && apt-get -y install --no-install-recommends <your-package-list-here>
+RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
+    && apt-get -y install --no-install-recommends librdkafka-dev pandoc
```

```suggestion
    && apt-get -y install --no-install-recommends librdkafka-dev pandoc
```
```suggestion
    && apt-get -y install --no-install-recommends pandoc
```

you should not need this dependency if you use the main branch where we updated confluent-kafka to a version which brings its needed libs packaged within.

---

Ok, I will remove the dependency then.